tool/rig/main.go

```diff
@@ -46,6 +47,13 @@ func run() error {
 		cupsertChangeset = Ref(cupsert.Flag("changeset", "name of the changeset").Short('c').Envar(changesetEnvVar).Required())
 		cupsertFile      = cupsert.Flag("file", "file with new resource spec").Short('f').Required().String()
 
+		cupsertConfigMap          = app.Command("configmap", "Upsert configmap in the context of a changeset")
+		cupsertConfigMapChangeset = Ref(cupsertConfigMap.Flag("changeset", "name of the changeset").Short('c').Envar(changesetEnvVar).Required())
+		cupsertConfigMapName      = cupsertConfigMap.Arg("name", "ConfigMap name").Required().String()
+		cupsertConfigMapNamespace = cupsertConfigMap.Flag("resource-namespace", "ConfigMap namespace").Default(rigging.DefaultNamespace).String()
+		cupsertConfigMapFiles     = cupsertConfigMap.Flag("from-file", "files or directories with contents").Strings()
```

Comment:
@a-palchikov the whole design is take from kubectl and is similar on purpose, so it is 1 to 1 mapping to the kubectl behavior